charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -107,10 +107,10 @@ spec:
           args:
             - "--metrics-addr=:8080"
             - "--enable-leader-election"
-            {{ if .Values.useWebhook }}
+            {{ if .Values.admissionWebhooks.enabled }}
             - "--use-webhook=true"
             - "--webhook-port={{ .Values.webhookService.port }}"
-            - "--webhook-cert-dir={{ .Values.certificate.mountPath }}"
+            - "--webhook-cert-dir=/etc/k8s-webhook-certs"
```

Comment:
why hard code this?

Reply:
yes, please add back, thanks, we need it in inner use case